modules/porous_flow/test/tests/fluids/tests

```diff
@@ -1,134 +1,145 @@
+    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in hours."
     issues = "#13154"
     design = 'PorousFlowSingleComponentFluid.md'
-  [../]
-  [./simple_fluid_yr]
+  []
+  [simple_fluid_yr]
     type = 'CSVDiff'
     input = 'simple_fluid_yr.i'
     csvdiff = 'simple_fluid_yr_out.csv'
     rel_err = 1.0E-5
     threading = '!pthreads'
-    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in years"
+    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in years."
     issues = "#13154"
     design = 'PorousFlowSingleComponentFluid.md'
-  [../]
-  [./simple_fluid_yr_MPa_C]
+  []
+  [simple_fluid_yr_MPa_C]
     type = 'CSVDiff'
     input = 'simple_fluid_yr_MPa_C.i'
     csvdiff = 'simple_fluid_yr_MPa_C_out.csv'
     rel_err = 1.0E-5
     threading = '!pthreads'
-    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in years, the pressure unit in MPa and the temperature unit in Celsius"
+    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in years, the pressure unit in MPa and the temperature unit in Celsius."
     issues = "#13154"
     design = 'PorousFlowSingleComponentFluid.md'
-  [../]
-  [./simple_fluid_yr_MPa_C_action]
+  []
+  [simple_fluid_yr_MPa_C_action]
     type = 'CSVDiff'
     input = 'simple_fluid_yr_MPa_C_action.i'
     prereq = simple_fluid_yr_MPa_C
     csvdiff = 'simple_fluid_yr_MPa_C_out.csv'
     rel_err = 1.0E-5
     threading = '!pthreads'
-    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in years, the pressure unit in MPa and the temperature unit in Celsius, and this functionality shall be available through the PorousFlowActions"
+    requirement = "PorousFlow shall be able to use equations of state from the FluidProperties module for a single-component fluid and specify the time unit in years, the pressure unit in MPa and the temperature unit in Celsius, and this functionality shall be available through the PorousFlowActions."
     issues = "#13154 #16841"
     design = 'PorousFlowSingleComponentFluid.md PorousFlowFullySaturated.md'
-  [../]
+  []
+  [multicomponent]
+    type = 'CSVDiff'
+    input = 'multicomponent.i'
+    csvdiff = 'brine1.csv'
+    rel_err = 1.0E-5
+    threading = '!pthreads'
+    prereq = brine1
+    requirement = "PorousFlow shall be able to use multicomponent equations of state from the FluidProperties module."
```

Comment:
Or perhaps it would be better for me to say that we should imagine that we write these requirements first in natural English and then we design the software and ensure we can meet the requirement.